scripts/app.js

```diff
@@ -9,15 +9,21 @@ function Project (opts) {
 }
 
 Project.prototype.toHtml = function () {
-  var $newProject = $('article.template').clone();
-  $newProject.attr('data-category', this.category);
-  $newProject.find('h2').text(this.title);
-  $newProject.find('time').html('about ' + parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000) + ' days ago');
-  $newProject.find('.project-body').append(this.image);
-  $newProject.find('.project-body').append(this.body);
-  $newProject.append('<hr>');
-  $newProject.removeClass();
-  return $newProject;
+  // var $newProject = $('article.template').clone();
+  // $newProject.attr('data-category', this.category);
+  // $newProject.find('h2').text(this.title);
+  // $newProject.find('time').html('about ' + parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000) + ' days ago');
+  // $newProject.find('.project-body').append(this.image);
+  // $newProject.find('.project-body').append(this.body);
+  // $newProject.append('<hr>');
+  // $newProject.removeClass();
+  // return $newProject;
```

Comment:
Remove the unneeded commented-out code. You can refer to it on previous branches if you need to.